### PR TITLE
feat: warn when messages were dropped

### PR DESCRIPTION
**Pull request by pauldambra**
We now record custom events within a recording when we have to drop messages because they were too large for Kafka

So we can show an error to folk when they view the recordings

(in this view we report 0 dropped messages but that's only so i could get the screenshot, really we don't show it if there are 0)

Screenshot 2024-06-28 at 13 53 12

### CODE REVIEW

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx [hunk 1]
```diff
@@ -58,6 +58,25 @@ function URLOrScreen({ lastUrl }: { lastUrl: string | undefined }): JSX.Element
...
 
+function PlayerWarningsRow(): JSX.Element | null {
+    const { messageTooLargeWarnings } = useValues(sessionRecordingPlayerLogic)
+    return messageTooLargeWarnings.length ? (
+        <div>
+            <LemonBanner
+                type="error"
+                action={{
+                    children: 'Learn more',
+                    to: 'https://posthog.com/docs/session-replay/troubleshooting#message-too-large-warning',
```
Comment (marandaneto) [inline frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:69]: #message-too-large-warning does not exist as an anchor?

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx [hunk 2]
```diff
@@ -58,6 +58,25 @@ function URLOrScreen({ lastUrl }: { lastUrl: string | undefined }): JSX.Element
...
+                    targetBlank: true,
+                }}
+            >
+                This session recording had {messageTooLargeWarnings.length} messages that were too large to be captured.{' '}
```
Comment (marandaneto) [inline frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:73]: Not sure if the wording `messages` here is ideal, snapshots is probably also very technical, but I wonder if they will understand what message here means

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx (hunk 1 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:69]: yep, coming soon :)

**Review by marandaneto — APPROVED** [review]

File: frontend/src/scenes/session-recordings/player/PlayerMeta.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/player/PlayerMeta.tsx:73]: yeah good point... something vague like "data" is probably the way to go 👍

**Review by daibhin — APPROVED** [review]

**Pull request merged**